README.md

```diff
@@ -1005,3 +1005,15 @@ the tests locally. These include:
   the root directory of the repo with the extension "xcconfig")
 * `UICATALOG_REAL_DEVICE` - path to the real device build of UICatalog, in case
   the npm installed one is not built for real device
+
+### Server Arguments
+
+Appium 2.0 Usage: `node . --driver-args='{"xcuitest": {[argNames]: [argValues]}}'`
+
+<expand_table>
+
+|Argument|Default|Description|Example|
+|----|-------|-----------|-------|
+
+|`"webkitDebugProxyPort"`|27753|Local port used for communication with ios-webkit-debug-proxy|`--driver-args='{"xcuitest": {"webkitDebugProxyPort": 27753}}'`|
```

Comment:
actually, we don't use the `ios-webkit-debug-proxy` component itself to communicate with the remote web debugger, but rather our own library.